docs/content/TreeView.mdx

```diff
@@ -10,63 +10,98 @@ description: A hierarchical list of items where nested items can be expanded and
 ### File tree navigation without directory links
```

Comment:
Could we get an example (and maybe a story?) showing how to use the `isExpanded` render prop to set different icons for expanded and collapsed states.

Reply:
Great idea 👍 Let me know if the docs I added make sense